app/auth/oauth_client.py

```diff
@@ -134,4 +134,4 @@ def _fix_expiration_time(self):
 
     def expires_soon(self):
         """Check if the client instance expires soon."""
-        return self._expires_at and self._expires_at < time.time() - 5
+        return self._expires_at and self._expires_at < time.time() + 5
```

Comment:
🙀

Reply:
"Vorzeichen sind Glückssache", as one of my profs always used to say...